src/constants/settingsTypes.ts

```diff
@@ -0,0 +1,5 @@
+const SHOW_HIDE_IMGS = 'show_hide_images';
```

Comment:
I link this approach, may be we can add a `TODO` comment to port more settings types here later.